go/libraries/doltcore/sqle/enginetest/dolt_queries.go

```diff
@@ -103,23 +103,23 @@ var DescribeTableAsOfScriptTest = queries.ScriptTest{
 		{
 			Query: "describe a as of @Commit1;",
 			Expected: []sql.Row{
-				{"pk", "int", "NO", "PRI", "", ""},
-				{"c1", "int", "YES", "", "", ""},
+				{"pk", "int", "NO", "PRI", "NULL", ""},
```

Comment:
Should these be `nil` instead of `"NULL"`? 

I recently had a bug where I was returning `"NULL"` from Dolt code for a row instead of `nil` and my tests were explicitly checking for `"NULL"`, so they passed, but when the code was used over the wire, that `"NULL"` string couldn't be converted to a timestamp (or whatever datatype it was) and it was a runtime error. Probably good to standardize on `nil` in tests to avoid problems that like being able to hide.

Reply:
actually... nevermind... on a closer look, I see that this is the default value column and we do need to indicate a default value (not the lack of any value), so `nil` doesn't make sense. I just hit this test failure as well, trying to get a GMS change into Dolt.